feat(serverless-contracts): add an optional arg in getHandler to disable input/output validation (#526)
swarmion/swarmion: user-docs/documentation/docs/how-to-guides/1-use-serverless-contracts/1-api-gateway.md

});
```

:::info

> By default, the getHandler feature will validate both input and output of your lambda. If you wish to disable one of those, you can use a second options argument int the `getHandler` feature:

Reviewer: ```suggestion
By default, the `getHandler` feature will validate both the input and the output of your lambda. If you wish to disable one of those, you can use the optional second argument in the `getHandler` feature:
```
